examples/create-next-app/open-next.config.ts

```diff
@@ -0,0 +1,15 @@
+import type { OpenNextConfig } from "@opennextjs/aws/types/open-next";
```

Comment:
here and in the other examples I get this:
![Screenshot 2024-11-04 at 10 17 21](https://github.com/user-attachments/assets/a1abe4c1-75ce-498e-87cd-941dcdf07a33)

To solve it I've installed `@opennextjs/aws` as a dev dependency of the app
and also updated the import to:
```suggestion
import type { OpenNextConfig } from "@opennextjs/aws/types/open-next.js";
```

Reply:
Touched based in a previous comment: this will ultimately goes away when we actually merge with aws